src/controllers/reservation.controller.js

```diff
@@ -0,0 +1,62 @@
+const reservationService = require('../services/reservation.service')
+
+exports.getReservation = (req, res) => {
+    const table = req.params.table
+    const tables = ["event", "dining", "spa"]
```

Comment:
I like how you handled having to reference different tables based on the service being requested, both here and in createReservation below. Nice and concise!